docs/v2.4/reference/restapi.md

```diff
@@ -0,0 +1,69 @@
+# Telepresence RESTful API server
+
+Telepresence can run a RESTful API server on the local host, both on the local workstation and in a pod that contains a `traffic-agent`. The server currently has two endpoints. The standard `healthz` endpoint and the `consume-here` endpoint.
+
+## Enabling the server
+The server is enabled by setting the `telepresenceAPI.port` to a valid port number in the [Telepresence Helm Chart](https://github.com/telepresenceio/telepresence/tree/release/v2/charts/telepresence). The values may be passed  explicitly to Helm during install, or configured using the [Telepresence Config](../config#restful-api-server) to impact an auto-install.
+
+## Querying the server
+On the cluster's side, it's the `traffic-agent` of potentially intercepted pods that runs the server. The server can be accessed using `http://localhost:<TELEPRESENCE_API_PORT>/<some endpoint>` from the application container. Telepresence ensures that the container has the `TELEPRESENCE_API_PORT` environment variable set when the `traffic-agent` is installed. On the workstation, it is the `user-daemon` that runs the server. It uses the `TELEPRESENCE_API_PORT` that is conveyed in the environment of the intercept. This means that the server can be accessed the exact same way locally, provided that the environment is propagated correctly to the interceptor process.
+
+## Endpoints
```

Comment:
i feel like it'd be a good idea to show the results of a `curl -v` on both endpoints, just to make it a little more graphic what to look for. For the `consume-here` it'd probably be good to show requests that return both `true` and `false`

I realize that in practice the user wouldn't be curling these endpoints manually, but I find `curl -v` to be a very immediately intuitive way to how to request an HTTP server and what it's responding with